pyEpiabm/pyEpiabm/routine/simulation.py

```diff
@@ -211,6 +289,42 @@ def write_to_file(self, time):
                 data["time"] = time
                 self.writer.write(data)
 
+    def write_to_ih_file(self, time, output_option: str):
+        """Records the infection history of the individual people
+        and writes these to file.
+
+        Parameters
+        ----------
+        time : float
+            Time of output data
+        output_option : str
+            Determines if you write data of infection status where \
+            output_option="status" and/or infectiousness where \
+            output_option="infectiousness"
+
+        """
+        if self.status_output:
+            if output_option == "status":
```

Comment:
Does anything happen if only one of these is true? It looks like you need both to be true (why so we need both?), in which case you can combine the two logical statements with `&` in one line

Reply:
Status_output sets up the writer and the file name for the specific csv, ready to be used if true. When this writer is set up, it will then call write_to_ih_file. We use the output_option as a way to differentiate the action that needs to be performed as for both types of csv created, they both call the same function.